tmva/sofie/test/TestCustomModelsFromONNX.cxx

```diff
@@ -2515,7 +2515,7 @@ TEST(ONNX, Equal){
    });
 
    TMVA_SOFIE_Equal::Session s("Equal_FromONNX.dat");
-   std::vector<bool> output = s.infer(input1.data(),input2.data());
```

Comment:
Did that fail to differentiate?

Reply:
No, I didn't even try to differentiate the models in the test. I'm solely focusing on the SBI usecase that we implement with LHCb. The reason why I changed this is because `std::vector<bool>` is not a good output type parameter. See:
  * https://github.com/root-project/root/pull/18302